README.md

```diff
@@ -119,6 +119,15 @@ bundle exec sidekiq -e production -q default -q mailers -d -L tmp/sidekiq.log` (
 
 This repository contains the [Code of Conduct](./CODE%20OF%20CONDUCT.md) of [CircuitVerse](https://circuitverse.org) Community.
 
+## CircuitVerse Community
+
+We would love to hear from you :smile:
+Here are links to our:
+
+[Gitter Channel](https://gitter.im/CircuitVerse/community)
+
+[Discord Channel](https://discord.gg/95x8H7b)
```

Comment:
Please mention that this is an unofficial fan communication channel. We absolutely don't want people landing to discord and DMing us for bug fixes or setup problems.

Reply:
Also mention @jbox144 for the initiative

Reply:
@armudgal Ok will do so. I think I got confused between GSoC and GSSoC. I am sorry for that

Reply:
@armudgal I think I would have to make another pr right?

Reply:
@creyente1897 Looks like it